src/libraries/System.IO.FileSystem.Watcher/ref/System.IO.FileSystem.Watcher.cs

```diff
@@ -15,10 +15,10 @@ public partial class ErrorEventArgs : System.EventArgs
     public delegate void ErrorEventHandler(object sender, System.IO.ErrorEventArgs e);
```

Comment:
NIT I thought event sender can be nullable

Reply:
This delegate type is associated with FileSystemWatcher, and FileSystemWatcher only ever invokes it with a non-null sender (it always passes in `this`), so while we could allow `sender` to be nullable, it's accurate to say that the intent is null is never passed here.